test/runtests.jl

```diff
@@ -102,7 +102,21 @@ using Test
 
         @test chain1[1].params == val
     end
+    
+    @testset "MALA" begin
+        using DiffResults, ForwardDiff, LinearAlgebra
```

Comment:
I'd prefer if all packages are loaded at rhe top of the script. Can you move this line to the other imports?